### PR TITLE
Initial add of sporks.json file to reflect the current live docs and also fix some errors

### PR DESCRIPTION
**Pull request by franklywatson**
Closes: #???

## Description

The newly added sporks.json is intended for automated use cases and reflect the contents of the current live doc https://docs.onflow.org/node-operation/past-sporks/. This commit also fixes some errors in the current docs

### CODE REVIEW

File: sporks.json [hunk 1]
```diff
@@ -0,0 +1,950 @@
+{
```
Comment (vishalchangrani) [inline sporks.json:1]: do we want both then? past-sporks.mdx and this file?

Comment (10thfloor) [reply sporks.json:1]: We could use this file to populate the docs pages. That might be handy.

**Review by vishalchangrani — APPROVED** [review]

File: sporks.json (hunk 1 above)
Comment (franklywatson) [reply sporks.json:1]: Good question Vishal. I guess it depends if we think it's best to share all the details in both places, or if we're keeping the .mdx file a subset.

Comment (Kay-Zee) [reply sporks.json:1]: I like the idea of using the json to generate the mdx file

**Review by Kay-Zee — APPROVED** [review]
I noticed there are still a lot of empty space, are there plans to backfill this data? e.g. the cadence tags, and the older access nodes?

**Comment by franklywatson** [comment]
> 

@Kay-Zee yes we should probably backfill the data that is missing, though it could take some time. Perhaps something we can outsource?

**Pull request merged**